### PR TITLE
Cloud account AWS methods and examples

### PR DESCRIPTION
**Pull request by talsheldon**

### CODE REVIEW

File: examples/cloudaccounts/azure/main.go [hunk 1]
```diff
@@ -14,7 +14,6 @@ func main() {
 	var req azure.CloudAccountRequest
 
 	req.Name = "test Azure cloud account"
-	req.Vendor = "Azure"
```
Comment (talsheldon) [inline examples/cloudaccounts/azure/main.go]: This is a fix (I forgot to remove it)

File: examples/cloudaccounts/aws/main.go [hunk 2]
```diff
@@ -46,4 +46,57 @@ func main() {
 	}
 
 	fmt.Printf("update name response type: %T\n Content: %+v\n", updateResponse, updateResponse)
+
+	// UpdateRegionConfig
+	desiredGroupBehavior := "FullManage"
+	updateConfigResponse, _, err := srv.UpdateRegionConfig(aws.CloudAccountUpdateRegionConfigRequest{
```
Comment (udidome9) [inline examples/cloudaccounts/aws/main.go]: ```updateConfigResponse``` should be ```updateRegionConfig```

File: examples/cloudaccounts/aws/main.go [hunk 3]
```diff
@@ -46,4 +46,57 @@ func main() {
...
+			NewGroupBehavior: desiredGroupBehavior,
+		},
+	})
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Printf("update region config response type: %T\n Content: %+v\n", updateConfigResponse, updateConfigResponse)
+
+	// Update credentials
```
Comment (udidome9) [inline examples/cloudaccounts/aws/main.go]: copy pasta

File: services/cloudaccounts/aws/aws.go [hunk 4]
```diff
@@ -91,6 +91,23 @@ type CloudAccountUpdateRegionConfigRequest struct {
 	} `json:"data,omitempty"`
 }
 
+type CloudAccountUpdateOrganizationalIdRequest struct {
```
Comment (udidome9) [inline services/cloudaccounts/aws/aws.go]: `CloudAccountUpdateOrganizationalIdRequest ` should be `CloudAccountUpdateOrganizationalIDRequest `

**Pull request merged**